src/framework/mlt_frame.c

```diff
@@ -334,7 +330,9 @@ mlt_deque mlt_frame_service_stack( mlt_frame self )
 
 int mlt_frame_set_image( mlt_frame self, uint8_t *image, int size, mlt_destructor destroy )
```

Comment:
This demonstrates how we can deprecate the old function to keep existing code mostly working until it is eventually converted.